test/test_data.rb

```diff
@@ -672,5 +672,208 @@ def make_paid_order(params={})
         :charge => make_charge,
       }).merge(params)
     end
+    
+    
+    
+    def country_spec_array
+      {
+        :object => "list",
+        :url => "/v1/country_specs",
+        :data => [
+          make_country_spec,
+          make_country_spec,
+          make_country_spec,
+        ]
+      }
+      
+    end
+    
+    def make_country_spec(params={})
+      {
+        :id=>"US",
+        :object=>"country_spec",
+        :supported_bank_account_currencies=> {:usd => ["US"]},
+        :supported_payment_currencies=>
+        ["usd",
```

Comment:
Mind just changing your multi-line arrays and hashes below to be like the other ones in this file? i.e. Instead of this:

``` ruby
[item1,
  item2,
  item3]

{:minimum=>{
  ...
  }}
```

Do this:

``` ruby
[
  item1,
  item2,
  item3
]

{
  :minimum => {
  }
}
```